<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Elpida::Application::BenchmarkRunConfigurationView</class>
 <widget class="QWidget" name="BenchmarkRunConfigurationView">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblRun">
     <property name="text">
      <string>Run it </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spnTimes">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lblTimes">
     <property name="text">
      <string>times.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCheckBox" name="chkUpload">
     <property name="text">
      <string>Upload results anonymously to elpida website.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="chkOpenResult">
     <property name="text">
      <string>Open result in browser after upload.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
